README.md

```diff
@@ -43,6 +43,26 @@ In the browser the balena SDK doesn't use the following dependencies:
 
 For the future we're looking at ways to automatically exclude these in downstream bundles. See [#254](https://github.com/balena-io/balena-sdk/issues/254) for more information.
 
+### Bundling with pkg
+
+Because the balena SDK includes builds for various ecmascript versions and the way that it loads the appropriate versions using [@balena/es-version](https://github.com/balena-io-modules/balena-es-version)
+packagers like [pkg](https://github.com/vercel/pkg) are not able to automatically detect which assets should be included in the build output. In such case you need to manually define a `pkg` section in
+your package.json, which will instruct `pkg` to bundle all the required assets:
+
+```json
+  "pkg": {
+    "scripts": [
+      "node_modules/balena-sdk/**/*.js"
+    ],
+    "assets": [
+      "node_modules/balena-pine/**/*",
+      "node_modules/pinejs-client-core/**/*"
+    ]
+  }
+```
+
+For more information, please refer to the respective (documentation from the `pkg` project)[https://github.com/vercel/pkg#config].
```

Comment:
Looks like the square brackets and the parentheses were swapped (markdown link syntax).